app/src/main/java/com/cornellappdev/score/components/UpcomingGameCard.kt

```diff
@@ -0,0 +1,161 @@
+            .clip(RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp))
+            .fillMaxWidth()
+            .background(
+                brush = Brush.linearGradient(
+                    colors = listOf(gradientColor1, gradientColor2)
+                )
+            )
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.spacedBy(32.dp, Alignment.CenterHorizontally),
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 36.dp)
+        ) {
+            Image(
+                painter = leftTeamLogo,
+                contentDescription = "Left Team Logo",
+                contentScale = ContentScale.FillBounds,
+                modifier = Modifier.size(60.dp)
+            )
+
+            Text(
+                text = "VS",
+                style = TextStyle(
+                    fontSize = 24.sp,
+                    fontFamily = poppinsFamily,
+                    fontWeight = FontWeight(700),
+                    fontStyle = FontStyle.Italic,
+                    color = Color(0xFFFFFFFF),
+                )
+            )
+
+            Image(
+                painter = rightTeamLogo,
+                contentDescription = "Right Team Logo",
+                modifier = Modifier.height(62.dp)
+            )
+        }
+    }
+}
+
+@Preview
+@Composable
+fun UpcomingGameHeaderPreview() {
+    UpcomingGameHeader(
+        leftTeamLogo = painterResource(R.drawable.cornell_logo),
+        rightTeamLogo = painterResource(R.drawable.penn_logo),
+        gradientColor1 = Color(0xE1A69F),
+        gradientColor2 = Color(0x011F5B) ,
+        modifier = Modifier
+    )
+}
+
+@Composable
+fun UpcomingGameCard(
+    leftTeamLogo: Painter,
+    rightTeamLogo: Painter,
+    team: String,
+    location: String,
+    genderIcon: Painter,
+    sportIcon: Painter,
+    date: String,
+    modifier: Modifier,
+    headerModifier:Modifier,
+    gradientColor1:Color,
+    gradientColor2:Color
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+    ) {
+
+        UpcomingGameHeader(
+            leftTeamLogo = leftTeamLogo,
+            rightTeamLogo = rightTeamLogo,
+            gradientColor1 = gradientColor1,
+            gradientColor2 = gradientColor2,
+            modifier = headerModifier
+        )
+
+        SportCard(
+            teamLogo = rightTeamLogo,
+            team = team,
+            date = date,
+            location = location,
+            genderIcon = genderIcon,
+            sportIcon = sportIcon,
+            topCornerRound = false,
+            modifier = modifier
+                .clip(RoundedCornerShape(topStart = 0.dp, topEnd = 0.dp, bottomStart = 16.dp, bottomEnd = 16.dp))
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun GameScheduleScreen() {
+    UpcomingGameCard(
+        leftTeamLogo = painterResource(R.drawable.cornell_logo),  // Cornell logo
```

Comment:
The preview here seems to pass in two gradient colors, but the screenshot you took seems to just have a white background. Is that an outdated screenshot?

Otherwise this preview looks great :-)